uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -86,7 +86,7 @@ def data_received(self, data):
                 self.transport.write(self.conn.send(err.event_hint))
                 self.transport.close()
             else:
-                self.handle_no_connect(events.CloseConnection())
+                self.handle_no_connect(events.CloseConnection(code=1007))
```

Comment:
```suggestion
                # Response with the "1002 Protocol Error" code.
                # See https://www.iana.org/assignments/websocket/websocket.xhtml#close-code-number
                self.handle_no_connect(events.CloseConnection(code=1002))
```

Reply:
That makes sense. Thanks! 🙇

I'll update later on.

Reply:
Cool.

Lazy option if you're happy with the suggestion. Just whack the big 'ol button.

![Screen Shot 2022-05-11 at 10 58 42](https://user-images.githubusercontent.com/647359/167824173-da1d590d-4794-4b15-96b2-9f4bd0cba3ae.png)

(You probs already know this, but mentioning it in case it's a bit of the GitHub UI you've not noticed yet. It's such a super helpful tool.)

Reply:
Yeah, but when I saw your suggestion, I thought it would be better if I add a test here. 😅